deploy/acm-policies/50-GENERATED-rosa-ingress-certificate.Policy.yaml

```diff
@@ -0,0 +1,93 @@
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+    annotations:
+        policy.open-cluster-management.io/categories: CM Configuration Management
+        policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+        policy.open-cluster-management.io/standards: NIST SP 800-53
+    name: rosa-ingress-certificate
+    namespace: openshift-acm-policies
+spec:
+    disabled: false
+    policy-templates:
+        - objectDefinition:
+            apiVersion: policy.open-cluster-management.io/v1
+            kind: ConfigurationPolicy
+            metadata:
+                name: rosa-ingress-certificate
+            spec:
+                evaluationInterval:
+                    compliant: 2h
+                    noncompliant: 45s
+                object-templates:
+                    - complianceType: musthave
+                      metadataComplianceType: musthave
+                      objectDefinition:
+                        apiVersion: v1
+                        stringData:
+                          tls.crt: |
+                            {{hub ( index ( lookup "v1" "Secret" "default" .ManagedClusterName ).data "tls.crt" ) | base64dec | autoindent hub}}
+                          tls.key: |
+                            {{hub ( index ( lookup "v1" "Secret" "default" .ManagedClusterName ).data "tls.key" ) | base64dec | autoindent hub}}
```

Comment:
If possible you don't want to specify `stringData` with policy if at all possible since the k8s api is not going to return it as stringdata so the compare in the config policy will always fail.  Here's a sample where a hub certificate is pushed to managed clusters without using stringData: https://github.com/open-cluster-management-io/policy-collection/blob/main/policygenerator/policy-sets/community/openshift-plus/input-sensor/policy-advanced-managed-cluster-security.yaml#L30-L39